1.9/usage/cli/command-reference/dcos-task/dcos-task-exec.md

```diff
@@ -0,0 +1,34 @@
+---
+post_title: dcos task exec
+menu_order: 0
+---
+
+# Description
+Launch a process inside of a task container.
+
+# Usage
+
+```bash
+dcos task exec [OPTION]
+```
```

Comment:
@joel-hamill should already have a bunch of examples from when we put together https://github.com/dcos/dcos-docs/blob/master/1.9/administration/debugging/quickstart.md

The same ones can be duplicated here.